src/scope/objects/repository.ts

```diff
@@ -115,9 +115,8 @@ export default class Repository {
     const refs = await this.listRefs();
     return Promise.all(refs.map(ref => this.load(ref)));
   }
-
-  // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
-  async listRefs(): Promise<Refs[]> {
+  // type any
+  async listRefs(): Promise<Array<any>> {
```

Comment:
@qballer , it needs to be `Ref[]`, it was probably a typo before (`Refs[]` with `s`).

Reply:
Cool, will change that.